tests/unit/utils/test_process.py

```diff
@@ -535,16 +531,42 @@ def test_process_list_cleanup(self):
         plist = salt.utils.process.SubprocessList()
         event = multiprocessing.Event()
         proc = multiprocessing.Process(target=event_target, args=[event])
+        proc2 = multiprocessing.Process(target=event_target, args=[event])
         proc.start()
+        proc2.start()
         plist.add(proc)
+        plist.add(proc2)
         assert proc in plist.processes
+        assert proc2 in plist.processes
         plist.cleanup()
         event.set()
         assert proc in plist.processes
+        assert proc2 in plist.processes
         self.wait_for_proc(proc)
+        self.wait_for_proc(proc2)
         assert not proc.is_alive()
+        assert not proc2.is_alive()
         plist.cleanup()
         assert proc not in plist.processes
+        assert proc2 not in plist.processes
+
+    @skipIf(True, "SLOWTEST skip")
```

Comment:
Let's add instead the slow test decorator?

Reply:
Sure! Thank you @s0undt3ch !

Reply:
I wonder how it could be missed. :shrug:. Fixed.